src/OpenTK/INativeWindow.cs

```diff
@@ -285,6 +285,7 @@ public interface INativeWindow : IDisposable
         //event EventHandler<MouseEventArgs> MouseClick;
         //event EventHandler<MouseEventArgs> MouseDoubleClick;
 
+        event EventHandler<OpenTK.Input.DropEventArgs> Drop;
```

Comment:
It's not necessarily a file, as I understand it - it could be image metadata from when a dragged image is dropped, for instance.